src/reset.c

```diff
@@ -47,12 +47,20 @@ static void _show_reset_label(bool status)
 void reset_reset(bool status)
 {
     keystore_lock();
+    uint8_t retries = 0, max_retries = 5;
+    bool sc_result = false;
 #if !defined(TESTING)
-    if (!securechip_update_keys()) {
+    while (!sc_result && retries++ < max_retries) {
+        sc_result = securechip_update_keys();
+    }
+    if (!sc_result) {
         Abort("Could not reset secure chip.");
     }
 #if APP_U2F == 1
-    if (!securechip_u2f_counter_set(0)) {
+    while (!sc_result && retries++ < max_retries) {
```

Comment:
I think this never executes because `sc_result` is `true` after `securechip_update_keys()` succeeds?

Reply:
i thought the `#if` cases were mutual exclusive, i missed this 😄